.buildkite/packaging.pipeline.yml

```diff
@@ -28,6 +28,7 @@ steps:
           - make beats-dashboards
         env:
           SNAPSHOT: true
+          DEV: true
```

Comment:
should we be explicit and add

```
SNAPSHOT: false
DEV: false
```

in the step below too? (Staging Dashboards)